platform.md

```diff
@@ -569,9 +693,10 @@ Usage:
       - All run-image layers SHALL be preserved
       - All run-image config values SHALL be preserved unless this conflicts with another requirement
     - MUST contain all buildpack-provided launch layers as determined by the [Buildpack Interface Specfication](buildpack.md)
-    - MUST contain a layer containing all buildpack-provided Software Bill of Materials (SBOM) files for `launch` as determined by the [Buildpack Interface Specfication](buildpack.md) if they are present
+    - MUST contain a layer containing all image extension-provided and buildpack-provided Software Bill of Materials (SBOM) files for `launch` as determined by the [Buildpack Interface Specfication](buildpack.md) if they are present
```

Comment:
This is a bit weird. If image extension SBOM files are included here, there could be collisions between image extension ids and buildpack ids... I am still not sure when image extensions would need to output SBOM files, because we have `genpkgs`. It would be nice to remove this if we can.

Reply:
Per latest discussion, we decided to take SBOM files out of the extension api for now. An extension could use a Dockerfile to write an SBOM file for a downloaded artifact if necessary.